src/webview/components/kdbDataSourceView.ts

```diff
@@ -214,24 +215,22 @@ export class KdbDataSourceView extends LitElement {
 
   private renderApiOptions(selected: string) {
     if (this.isInsights && this.isMetaLoaded) {
-      return html`
-        ${this.insightsMeta.api
-          .filter(
-            (api) => api.api === ".kxi.getData" || !api.api.startsWith(".kxi.")
-          )
-          .map((api) => {
-            const value =
-              api.api === ".kxi.getData"
-                ? api.api.replace(".kxi.", "")
-                : api.api;
-
-            return html`
-              <vscode-option value="${value}" ?selected="${value === selected}"
-                >${value}</vscode-option
-              >
-            `;
-          })}
-      `;
+      return this.insightsMeta.api
```

Comment:
why this change? if we remove it the html response extensions that make more easy/readable to maintain it later

Reply:
Outer html``is superfluous as you may check from the tests.